Source/Core/Core/Movie.h

```diff
@@ -75,8 +75,8 @@ struct DTMHeader
   std::array<char, 6> gameID;  // The Game ID
   bool bWii;                   // Wii game
 
-  u8 controllers;  // Controllers plugged in (from least to most significant,
-                   // the bits are GC controllers 1-4 and Wiimotes 1-4)
+  u16 controllers;  // Controllers plugged in (from least to most significant,
+                    // the bits are GC controllers 1-4, Wiimotes 1-4 and GBAs 1-4)
```

Comment:
This change breaks DTM comparability, as it causes other fields to be offset by one byte. Any new data you want to add needs to be placed in a byte that currently is reserved (and preferably `reserved` instead of `reserved3` since `reserved3` used to have a different meaning).

Reply:
Maybe `reserved3` should've been documented then :)
But thats for another PR.